packages/language-server/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "@neo4j-cypher/language-support": "2.0.0-next.5",
+    "@neo4j-cypher/schema-poller": "2.0.0-next.5",
```

Comment:
This is not going to work @OskarDamkjaer. Basically there was a reason why these dependencies were listed in the `turbo.json` instead. If you declare this here and the package `db-poller` is not published to `npm`, this won't work.